src/mavlink_stream.c

```diff
@@ -24,6 +25,11 @@ static void mavlink_send_heartbeat(void)
                              MAV_STATE_STANDBY);
 }
 
+static uint32_t time_mavlink_send_message(void* f)
```

Comment:
This is new to me; are we actually using this? If not perhaps we should get rid of it